assets/app/views/deployments.html

```diff
@@ -10,14 +10,112 @@
           <em>{{emptyMessage}}</em>
         </div>
       </div>      
-      <div style="margin-bottom: 10px;" ng-repeat="deployment in deployments">
-        <h3>{{deployment.metadata.annotations.deploymentConfig}} <span class="small">({{deployment.metadata.name}})</span></h3>
-        <div>Created: <relative-timestamp timestamp="deployment.metadata.creationTimestamp"></relative-timestamp></div>
-        <div>Status: {{deployment.metadata.annotations.deploymentStatus}}</div>
-        <div>Version: {{deployment.metadata.annotations.deploymentVersion}}</div>
-        <div>Replicas: {{deployment.spec.replicas}}</div>
-        <pod-template ng-init="podTemplate = deployment.spec.template"></pod-template>
+      <div class="tile" ng-repeat="(deploymentConfigName, deploymentConfig) in deploymentConfigs">
+        <div> 
+          <h2>{{deploymentConfigName}}</h2>
+          <dl class="dl-horizontal left indent">
+          <div>
+              <dt>Created:</dt>
+              <dd><relative-timestamp timestamp="deploymentConfig.metadata.creationTimestamp"></relative-timestamp></dd>
+          </div>
+          <div>
+              <dt>Labels:</dt>
+              <dd>{{deploymentConfig.metadata.labels}}</dd>
```

Comment:
can we render the labels as key=value lines, rather than JSON?
```
<dd>
  <div ng-repeat="(labelKey, labelValue) in deploymentConfig.metadata.labels">{{labelKey}}={{labelValue}}</div>
</dd>
```